src/slack/vacuumlabs/constants.js

```diff
@@ -11,102 +11,162 @@ Do you want to order 3 Cool Coolers and 2 Cool Smartphones? Just tell me so:
 
 Happy shopping!
 
-PS: Feel free to contribute at https://github.com/vacuumlabs/eshop-api`
+PS: Feel free to contribute at https://github.com/vacuumlabs/eshop-api`;
+
+const SPINOFFS = [
+  "AdaLite",
+  "ReactiveConf",
+  "Sparring",
+  "Trama",
+  "Verdicto",
+  "Daylight",
+  "Rychlotest-covid",
+]
 
 const CITIES_OPTIONS = {
-  bratislava: 'Bratislava',
-  kosice: 'Košice',
-  presov: 'Prešov',
-  praha: 'Praha',
-  brno: 'Brno',
-  budapest: 'Budapest',
+  bratislava: "Bratislava",
```

Comment:
1. separate the formatting commit from the business logic changes
2. create a new task and PR for adding prettier config to the project